cjson/PSPBUILD

```diff
@@ -0,0 +1,30 @@
+pkgname=cJSON
+pkgver=1.7.16
+pkgrel=1
+pkgdesc="Ultralightweight JSON parser in ANSI C."
+arch=('mips')
+url="https://github.com/DaveGamble/cJSON/"
+license=('MIT license')
+depends=()
+makedepends=()
+optdepends=()
+source=("https://github.com/DaveGamble/cJSON/archive/refs/tags/v1.7.16.tar.gz")
+sha256sums=('SKIP')
```

Comment:
Please add a checksum

Reply:
I added the checksum. 👍 
Note: I wasn't sure if Github actually keeps the same file or "regenerates" the .tgz or .zip release file.

Reply:
Thanks! It should